[0.223s] Invoking command in '/home/devpc/Desktop/quad_ws/build/quad_interfaces': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/home/devpc/Desktop/quad_ws/install/quad:/opt/ros/foxy /usr/bin/cmake --build /home/devpc/Desktop/quad_ws/build/quad_interfaces -- -j8 -l8
[0.247s] -- Found ament_cmake: 0.9.8 (/opt/ros/foxy/share/ament_cmake/cmake)
[0.266s] -- Found PythonInterp: /usr/bin/python3 (found suitable version "3.8.10", minimum required is "3") 
[0.266s] -- Using PYTHON_EXECUTABLE: /usr/bin/python3
[0.440s] -- Found ament_lint_auto: 0.9.6 (/opt/ros/foxy/share/ament_lint_auto/cmake)
[0.502s] -- Found rosidl_default_generators: 1.0.1 (/opt/ros/foxy/share/rosidl_default_generators/cmake)
[0.505s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.512s] -- Found rosidl_adapter: 1.2.1 (/opt/ros/foxy/share/rosidl_adapter/cmake)
[0.653s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.720s] -- Found PythonInterp: /usr/bin/python3 (found version "3.8.10") 
[0.904s] CMake Error at /opt/ros/foxy/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:60 (message):
[0.904s]   execute_process(/usr/bin/python3 -m rosidl_adapter --package-name
[0.904s]   quad_interfaces --arguments-file
[0.904s]   /home/devpc/Desktop/quad_ws/build/quad_interfaces/rosidl_adapter__arguments__quad_interfaces.json
[0.904s]   --output-dir
[0.904s]   /home/devpc/Desktop/quad_ws/build/quad_interfaces/rosidl_adapter/quad_interfaces
[0.904s]   --output-file
[0.905s]   /home/devpc/Desktop/quad_ws/build/quad_interfaces/rosidl_adapter/quad_interfaces.idls)
[0.905s]   returned error code 1:
[0.905s] 
[0.905s]   Error processing 'bool emergencyStop' of 'quad_interfaces/EmergencyStop':
[0.905s]   ''emergencyStop' is an invalid field name.  It should have the pattern
[0.905s]   '^(?!.*__)(?!.*_$)[a-z][a-z0-9_]*$''
[0.905s] 
[0.905s]   Traceback (most recent call last):
[0.905s] 
[0.905s]     File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
[0.906s]       return _run_code(code, main_globals, None,
[0.906s]     File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
[0.906s]       exec(code, run_globals)
[0.906s]     File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/__main__.py", line 19, in <module>
[0.906s]       sys.exit(main())
[0.906s]     File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/main.py", line 53, in main
[0.906s]       abs_idl_file = convert_to_idl(
[0.906s]     File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/__init__.py", line 19, in convert_to_idl
[0.906s]       return convert_msg_to_idl(
[0.906s]     File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/msg/__init__.py", line 28, in convert_msg_to_idl
[0.906s]       msg = parse_message_string(package_name, input_file.stem, content)
[0.907s]     File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/parser.py", line 515, in parse_message_string
[0.907s]       fields.append(Field(
[0.907s]     File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/parser.py", line 353, in __init__
[0.907s]       raise NameError(
[0.907s] 
[0.907s]   NameError: 'emergencyStop' is an invalid field name.  It should have the
[0.907s]   pattern '^(?!.*__)(?!.*_$)[a-z][a-z0-9_]*$'
[0.907s] 
[0.907s] Call Stack (most recent call first):
[0.907s]   /opt/ros/foxy/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[0.907s]   CMakeLists.txt:37 (rosidl_generate_interfaces)
[0.907s] 
[0.908s] 
[0.908s] -- Configuring incomplete, errors occurred!
[0.908s] See also "/home/devpc/Desktop/quad_ws/build/quad_interfaces/CMakeFiles/CMakeOutput.log".
[0.910s] make: *** [Makefile:748: cmake_check_build_system] Error 1
[0.912s] Invoked command in '/home/devpc/Desktop/quad_ws/build/quad_interfaces' returned '2': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/home/devpc/Desktop/quad_ws/install/quad:/opt/ros/foxy /usr/bin/cmake --build /home/devpc/Desktop/quad_ws/build/quad_interfaces -- -j8 -l8
